release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -165,6 +165,20 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-13-NUMA-scheduler-ga"]
+==== NUMA-aware scheduling with the NUMA Resources Operator is generally available
+
+With {product-title} {product-version}, NUMA-aware scheduling with the NUMA Resources Operator is generally available. This was a Technology Preview feature in {product-title} 4.10, 4.11, and 4.12. 
+
+The NUMA Resources Operator deploys a NUMA-aware secondary scheduler that makes scheduling decisions for workloads based on a complete picture of available NUMA zones in clusters. This ensures that latency-sensitive workloads are processed in a single NUMA zone for maximum efficiency and performance. 
+
+This update also adds the following features:
+
+* Fine-tuning of API polling for NUMA resource reports
+* Configuration options at the node group level for the node topology exporter
+
```

Comment:
@ffromani - are these two features introduced for versions 4.13 and later? Or are they available for earlier versions also?

Reply:
These features will be backported in the 4.12 Z stream. I don't have the version handy atm.